model/change_point_detection.go

```diff
@@ -126,6 +128,17 @@ type PerformanceData struct {
 	Data                []MeasurementData         `bson:"data"`
 }
 
+type ChangePointWithPerformanceData struct {
+	PerformanceResultSeriesID `bson:",inline"`
+	ChangePoint               ChangePoint `bson:"change_point"`
+	PerfResultId              string      `bson:"perf_result_id" json:"perf_result_id"`
```

Comment:
ID

Reply:
Why ID? This is for the performance result, not the change point itself.

Reply:
sorry PerfResultID rather just doing s/Id/ID/